packages/components/pie-notification/src/index.ts

```diff
@@ -96,9 +131,12 @@ export class PieNotification extends LitElement implements NotificationProps {
      *
      * @private
      */
-    private renderFooter () {
+    private renderFooter (leadingAction: ActionProps, supportingAction?: ActionProps) {
         return html`
-            <footer class="${componentClass}-footer"></footer>
+            <footer class="${componentClass}-footer" data-test-id="${componentSelector}-footer" is-compact="${this.isCompact}" is-stacked="${this.hasStackedActions && !this.isCompact}">
```

Comment:
we use camelCasing for custom props

```suggestion
            <footer class="${componentClass}-footer" data-test-id="${componentSelector}-footer" isCompact="${this.isCompact}" isStacked="${this.hasStackedActions && !this.isCompact}">
```

Reply:
also i think we should use [nothing](https://lit.dev/docs/api/templates/#nothing) is isStacked as it provides a consistent behavior between various expression binding contexts.

```suggestion
            <footer class="${componentClass}-footer" data-test-id="${componentSelector}-footer" is-compact="${this.isCompact}" is-stacked="${this.hasStackedActions && !this.isCompact ? true : nothing}">
```

Reply:
nit: a small suggestion, we don't need to pass the `leadingAction` and supportingAction as args to `renderFooter` since they're still part of the same class and can be accessed directly using the `this` keyword. 

the same thing applies for `renderNotificationHeading` and other a few places where we pass args around. happy to address that in a follow up PR if you find it a bit clearer.

Reply:
I found a better way to use camelCasing and avoiding ternaries by writing the custom props this way:
```
<div ?isCompact="${this.isCompact}" ?isStacked="${this.hasStackedActions && !this.isCompact}">
```
About your suggestion of removing args, I would like to bring this topic to the rest of the team.